Gem/Code/Source/Automation/ScriptReporter.h

```diff
@@ -246,6 +249,7 @@ namespace AtomSampleViewer
         DisplayOption m_displayOption = DisplayOption::AllResults;
         bool m_forceShowUpdateButtons = false; //< By default, the "Update" buttons are visible only for failed screenshots. This forces them to be visible.
         AZStd::string m_officialBaselineSourceFolder; //< Used for updating official baseline screenshots
+        AZStd::string m_exportTestResultsFolder; //< Used for storing exported test results
```

Comment:
This could just be a local variable since it looks like you're recomputing it based on the project path each time anyways